ui-admin/src/study/StudySettings.tsx

```diff
@@ -63,10 +62,18 @@ export function StudyEnvConfigView({ studyEnvContext, portalContext }:
           onChange={e => updateConfig('password', e.target.value)}/>
       </label>
     </div>
+    <div>
+      <label className="form-label">
+        accepting enrollment <InfoPopup content={`Uncheck this to hide the study from participants who have not
+        yet joined and prevent any further enrollments.`}/>
+        <input type="checkbox" checked={config.passwordProtected} className="ms-2"
```

Comment:
```suggestion
        <input type="checkbox" checked={config.acceptingEnrollment} className="ms-2"
```

I assume this was a copy/paste leftover 🙂